MAINTAINERS.yml

```diff
@@ -458,22 +458,23 @@ Display drivers:
   labels:
     - "area: Display"
 
-Documentation:
+Documentation Infrastructure:
   status: maintained
   maintainers:
     - carlescufi
   collaborators:
     - nashif
-    - utzig
     - gmarull
   files:
-    - doc/
-    - doc/_static/
-    - doc/_templates/
     - doc/_doxygen/
+    - doc/_extensions/
     - doc/_scripts/
-    - README.rst
+    - doc/_static/
+    - doc/_templates/
```

Comment:
to simplify this a bit, should we consider `doc/_*/` "infrastructure"?

Reply:
done, nice suggestion